compiler/src/iree/compiler/Codegen/Dialect/GPU/TargetUtils/ConfigUtils.cpp

```diff
@@ -249,22 +249,8 @@ LogicalResult setTileAndFuseLoweringConfig(IREE::GPU::TargetAttr target,
 
   // Configurations we need to decide.
   std::array<int64_t, 3> workgroupSize;
```

Comment:
While we are at it, could you also value-initialize this so that there are no undefined values biting us later?
```suggestion
  std::array<int64_t, 3> workgroupSize = {};
```